internal/fuse/dir.go

```diff
@@ -170,6 +181,10 @@ func (d *dir) ReadDirAll(ctx context.Context) ([]fuse.Dirent, error) {
 }
 
 func (d *dir) Lookup(ctx context.Context, name string) (fs.Node, error) {
+	err := d.Open()
+	if err != nil {
+		return nil, err
+	}
 	debug.Log("Lookup(%v)", name)
```

Comment:
please move the `debug.Log` call above `d.Open()`

Reply:
is changed.